src/pages/index.js

```diff
@@ -23,70 +31,121 @@ const addNewCardButton = document.querySelector(".profile__add-button");
+      },
+      ".cards__list"
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => console.error(err));
 
-const previewImageModal = document.querySelector("#modal-preview");
+const userInfo = new UserInfo({
+  nameSelector: ".profile__title",
+  descriptionSelector: ".profile__description",
+  avatarSelector: ".profile__image",
+});
 
-const previewModalCloseButton =
-  previewImageModal.querySelector(".modal__close");
+api
+  .getUsers()
+  .then((userData) => {
+    userInfo.setUserInfo({
+      name: userData.name,
+      description: userData.about,
+    });
+    userInfo.setAvatar(userData.avatar);
+  })
+  .catch((err) => console.error(err));
 
-/*--------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
-/*                                                                                                                                                                          */
-/*                                                                     Event Handlers                                                                                                                                                 */
-/*                                                                                                                                                                          */
-/*--------------------------------------------------------------------------------------------------------------------------------------------------------------------------*/
 function handleProfileEditSubmit(formValues) {
   const nameInput = formValues.title;
   const descriptionInput = formValues.description;
-  userInfo.setUserInfo({
-    name: nameInput,
-    description: descriptionInput,
-  });
-  profileModal.close();
-}
 
-const userInfo = new UserInfo({
-  nameSelector: ".profile__title",
-  descriptionSelector: ".profile__description",
-});
+  profileModal.setLoading(true);
+
+  api
+    .editUserInfo(nameInput, descriptionInput)
+    .then(() => {
+      userInfo.setUserInfo({
+        name: nameInput,
+        description: descriptionInput,
+      });
+      profileModal.close();
+    })
+    .finally(() => {
+      profileModal.setLoading(false);
+    });
+}
 
 function handleAddCardSubmit(inputValues) {
   const cardData = {
     name: inputValues.title,
     link: inputValues.url,
   };
 
-  const cardView = createCard(cardData);
-
-  cardSection.addItem(cardView);
+  addModal.setLoading(true);
+  api
+    .addCard(cardData)
+    .then((res) => {
+      cardSection.addItem(createCard(res));
+      addModal.close();
+      addCardFormElement.reset();
+      addCardValidator.disableButton();
+    })
+    .finally(() => {
+      addModal.setLoading(false);
+    });
+}
 
-  addModal.close();
-  addCardFormElement.reset();
-  addCardValidator.disableButton();
+function handleLikeClick(card) {
+  if (card._isLiked) {
```

Comment:
`_isLiked`


Please, delete the **low dash** (`underscore`) from methods/fields if you want to use them **outside** a class because the `low dash` shows a **private** field which is used only inside the class.